layouts/partials/sections/schedule-short.html

```diff
@@ -0,0 +1,20 @@
+<!-- short schedule section started -->
+<section class="schedule-section pb-5 pb-md-0" id="schedule">
+    <div class="container">
+        <div class="row">
+            <div class="col-md-12 text-center pt-md-5">
+                <h1 class="schedule-head text-white fw-bold pt-5" data-aos="fade-left" data-aos-duration="5000">
+                    Schedule</h1>
+                    <p class="schedule-para text-grey pt-md-3">PyCon India 2021 schedule has been released!</p>
+                <a href="/2021/schedule" class="yellow-text white-hovertext text-decoration-none fw-bold">Check out the schedule <i
```

Comment:
Maybe you can add `schedule/` instead of `/2021/schedule`. That helps keep it relative to whatever the root URL is set in hugo.